features/step_definitions/web_steps.rb

```diff
@@ -97,6 +114,12 @@ def with_scope(locator)
   end
 end
 
+Then /^visiting "([^"]*)" it should fail with an error$/ do |path|
```

Comment:
There's a stray "it" in here -- should be just `visiting "([^"]*)" should fail with an error`